x/ecocredit/client/testsuite/suite.go

```diff
@@ -0,0 +1,150 @@
+package testsuite
+
+import (
+	"github.com/stretchr/testify/suite"
+	dbm "github.com/tendermint/tm-db"
+
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/orm/model/ormdb"
+	"github.com/cosmos/cosmos-sdk/orm/model/ormtable"
+	"github.com/cosmos/cosmos-sdk/orm/types/ormjson"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	marketApi "github.com/regen-network/regen-ledger/api/regen/ecocredit/marketplace/v1"
+	api "github.com/regen-network/regen-ledger/api/regen/ecocredit/v1"
+	"github.com/regen-network/regen-ledger/types/testutil/network"
+	"github.com/regen-network/regen-ledger/x/ecocredit"
+	"github.com/regen-network/regen-ledger/x/ecocredit/core"
+)
+
+type IntegrationTestSuite struct {
+	suite.Suite
+
+	cfg     network.Config
+	network *network.Network
+	val     *network.Validator
+
+	// test accounts
+	addr1 sdk.AccAddress
+	addr  sdk.AccAddress // TODO: addr2
```

Comment:
is this still a todo?

Reply:
I was hoping to limit the scope of this pull request and further refine the integration tests as a whole. I can remove the todo or we can leave it for now. Also related to #922.

Reply:
ah ok. i think adding the issue link in the comment for context would help 👍🏻

Reply:
Good call. https://github.com/regen-network/regen-ledger/pull/1116/commits/168c6130177f6e2bf6b89f704c4e3c16f01aa087